docs/Style_Guidelines/ux_fonts.md

```diff
@@ -31,7 +31,7 @@ We recommend that you use a **sans serif font family** in your extension. Here,
 &nbsp;
 
 #### Benton Sans
-Benton Sans Book is the default font for the Tableau UI elements. <i>Learn more about Tableau font colors at **[Color](6 - Color.md)**.</i>
+Benton Sans Book is the default font for the Tableau UI elements. <i>Learn more about Tableau font colors at **[Color](ux_fonts.md)**.</i>
```

Comment:
@bcantoni should this point to `ux_color.md`

Reply:
Good catch! I must have gotten a little too excited here.

Reply:
Updated with this change.